packages/admin-theme/src/getShadows.ts

```diff
@@ -0,0 +1,29 @@
+import { Shadows } from "@material-ui/core/styles/shadows";
+
+export default (): Shadows => [
+    "none",
+    "0 0 8px 0 rgba(0,0,0,0.10)",
+    "0 0 20px 0 rgba(0,0,0,0.10)",
+    "none",
+    "0 0 40px 0 rgba(0,0,0,0.10)",
+    "none",
+    "none",
+    "none",
+    "none",
+    "none",
+    "none",
+    "none",
+    "none",
+    "none",
+    "none",
+    "none",
+    "none",
+    "none",
+    "none",
+    "none",
+    "none",
+    "none",
+    "none",
+    "none",
+    "none",
```

Comment:
FYI: All the `"none"`'s are intentional. 

The material-ui `Shadows` interface requires 25 types of shadows, our comet-design only has a value for "100", "200" and "400": https://zeroheight.com/5d7649959/p/176f36-layer-styles/t/277bd1